easybuild/easyconfigs/i/IPython/IPython-4.0.0-intel-2015a-Python-2.7.10.eb

```diff
@@ -0,0 +1,74 @@
+easyblock = 'Bundle'
+
+name = 'IPython'
+version = '4.0.0'
+
+homepage = 'http://ipython.org/index.html'
+description = """IPython provides a rich architecture for interactive computing with:
+ Powerful interactive shells (terminal and Qt-based).
+ A browser-based notebook with support for code, text, mathematical expressions, inline plots and other rich media.
+ Support for interactive data visualization and use of GUI toolkits.
+ Flexible, embeddable interpreters to load into your own projects.
+ Easy to use, high performance tools for parallel computing."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+python = 'Python'
+pyver = '2.7.10'
+pyshortver = '.'.join(pyver.split('.')[:2])
+versionsuffix = '-%s-%s' % (python, pyver)
+
+dependencies = [
+    (python, pyver),
+]
+
+# this is a bundle of Python packages
+# override extensions sanity check for IPython, importing a Python module doesn't work there
+exts_defaultclass = 'PythonPackage'
+exts_filter = ('python -c "import %(ext_name)s" || ipython -h', '')
+
+exts_list = [
+    ('traitlets', '4.0.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/t/traitlets/'],
+    }),
+    ('ipython_genutils', '0.1.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/i/ipython_genutils/'],
+    }),
+    ('pexpect', '3.3', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/pexpect/'],
+    }),
+    ('path.py', '7.6', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/path.py/'],
+    }),
+    ('pickleshare', '0.5', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/pickleshare/'],
+    }),
+    ('simplegeneric', '0.8.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/s/simplegeneric/'],
+        'source_tmpl': '%(name)s-%(version)s.zip',
+    }),
+    ('requests', '2.7.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/r/requests/'],
+    }),
+    ('testpath', '0.2', {
```

Comment:
testpath can't be installed using `PythonPackage` (yet), since it's only available as a Python wheel from PyPI, and there's no `setup.py` available in the GitHub repo...